config/runner/toolkit.yml

```diff
@@ -15,6 +15,12 @@ toolkit:
     asda_source: 'reference'
     myloader: '/scripts/importdb.sh'
     mydumper: '/scripts/backupdb.sh'
+  forbidden_files:
+    - name: ".env"
+      error: ".env file detected!\n In order to avoid loading .env file in ACC or PROD environments, please rename it to something like .env.dist\n The file belongs to your application. You should guarantee that is not being called into any point of your Drupal site. If you are only calling it in docker-compose.yml, it's recommended to drop entirely the file and declare the environment variables straight into docker-composer.yml file."
```

Comment:
Hi @ozyrys , the correct filename is `docker-compose.yml`

Reply:
Hi @joaocsilva it's fixed.